contracts/contracts/FleekERC721.sol

```diff
@@ -54,6 +67,7 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl {
         mapping(uint256 => Build) builds; // Mapping to build details for each build number
         string logo;
         uint24 color; // Color of the nft
+        bool accessPointAutoApprovalSettings; // Settings for approving new access points automatically
```

Comment:
```suggestion
        bool autoApproveAccessPoint; // Settings for approving new access points automatically
```
Thinking we do that or we can make a struct for Settings and put this one under it (assuming we can nest structs in Solidity).